src/main/java/de/drachir000/utils/config/SimpleConfigLib.java

```diff
@@ -42,4 +44,81 @@ public static Configuration buildConfiguration(String source) throws JSONExcepti
 		return buildConfiguration(new JSONObject(source));
 	}
 	
+	/**
+	 * Saves a {@link Configuration} to a {@link File}.
+	 *
+	 * @param configuration the {@link Configuration} to be saved
+	 * @param file          the {@link File} to save the {@link Configuration} to
+	 * @throws IOException if an I/O error occurs while writing to the file
+	 * @since 1.3
+	 */
+	public static void save(Configuration configuration, File file) throws IOException {
+		
+		BufferedWriter writer = new BufferedWriter(new FileWriter(file));
+		writer.write(configuration.toString());
+		
+		writer.flush();
+		writer.close();
+		
+	}
+	
+	/**
+	 * Saves a {@link Configuration} to a {@link File}.
+	 *
+	 * @param configuration        the {@link Configuration} to be saved
+	 * @param file                 the {@link File} to save the {@link Configuration} to
+	 * @param encodeUnknownObjects represents whether to encode unknown objects. If this is set to true, any {@link Serializable}
+	 *                             object, which is not recognized and hence would be converted into a String for the result, will be
+	 *                             serialized and Base64 encoded. Such objects can then be fetched using {@link Configuration#getEncoded(String)}.
+	 *                             This setting does not modify this Object, only the manner these objects are stored within the File by this method.
+	 * @throws IOException if an I/O error occurs while writing to the file
+	 * @since 1.3
+	 */
+	public static void save(Configuration configuration, File file, boolean encodeUnknownObjects) throws IOException {
```

Comment:
**First:** I belive it would be best to reuse the boolean value and then just call save(..., false) in the other save method.
**Second:** Better add a test case for this

Reply:
Implemented first request here: https://github.com/Drachir000/SimpleConfigLib/commit/02cbd50e1594c5738d984c1eb7b2ff0edf721ba7

Reply:
Implemented second request here: https://github.com/Drachir000/SimpleConfigLib/commit/91d95e7d5e81eb038488fa37a76ef73fab9db450